test/e2e/density.go

```diff
@@ -250,6 +250,36 @@ var _ = Describe("Density [Skipped]", func() {
 			stop := make(chan struct{})
 			go controller.Run(stop)
 
+			// Create a listener for api updates
+			updateCount := 0
+			label := labels.SelectorFromSet(labels.Set(map[string]string{"name": RCName}))
+			_, updateController := controllerframework.NewInformer(
+				&cache.ListWatch{
+					ListFunc: func(options api.ListOptions) (runtime.Object, error) {
+						options.LabelSelector = label
+						return c.Pods(ns).List(options)
+					},
+					WatchFunc: func(options api.ListOptions) (watch.Interface, error) {
+						options.LabelSelector = label
+						return c.Pods(ns).Watch(options)
+					},
+				},
+				&api.Pod{},
+				0,
+				controllerframework.ResourceEventHandlerFuncs{
+					AddFunc: func(_ interface{}) {
+						updateCount++
+					},
+					UpdateFunc: func(_, _ interface{}) {
+						updateCount++
+					},
+					DeleteFunc: func(_ interface{}) {
+						updateCount++
+					},
+				},
```

Comment:
I should remove them to just count "Update" count, it should be more meaningful for status manager. :)